eclair-core/src/main/scala/fr/acinq/eclair/db/pg/PgNetworkDb.scala

```diff
@@ -27,7 +27,7 @@ import fr.acinq.eclair.{RealShortChannelId, ShortChannelId}
 import grizzled.slf4j.Logging
 import scodec.bits.BitVector
 
-import java.sql.{Connection, Statement}
+import java.sql.{Connection, ResultSet, Statement}
```

Comment:
In theory we should update the db version and delete the old `network.pruned_channels` update I'm afraid

Reply:
I hesitated, it doesn't hurt to keep the unused table around though and remove it once we have a real DB update to make, to minimize the number of version changes?